src/lib/components/chat/MessageInput/VoiceRecording.svelte

```
@@ -38,24 +53,63 @@
 		return `${minutes}:${formattedSeconds}`;
 	};
 
-	let stream;
-	let speechRecognition;
+	let stream: MediaStream | null = null;
+
+	interface SpeechRecognition extends EventTarget {
+		continuous: boolean;
+		start(): void;
+		stop(): void;
+		onresult: (event: SpeechRecognitionEvent) => void;
```

ESLint

[error] 'SpeechRecognitionEvent' is not defined.

(no-undef)